README.md

```diff
@@ -85,16 +85,17 @@ them below.
 
 #### Authentication Support
 
-Visit the authentication config in your project and enable the `Anonymous` 
+Visit the authentication config in your project and enable the `Anonymous`
 sign-in provider to complete your project config.
 
 #### Automated Setup
 
-The remainder of the test setup can be done by running the following command at
+The remainder of the test setup can be done by running the following commands at
 the root of the package:
 
 ```bash
 yarn test:setup
+yarn build
```

Comment:
Actually, doesn't this belong with Running the tests?

`yarn test:setup` is a one-time thing, but `yarn build` is something you do every time you've made changes, no?

Reply:
Correct. `yarn build` doesn't belong here. 
Can you please add a section below Install Dependencies, something like:
### Build The SDK
Once you installed dependencies, you can build the entire SDK by running the following at the root of the SDK:

```bash
$yarn build
```